Examples/tvOSMultiplePlayersTemplate/tvOSMultiplePlayersTemplate/ViewController.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  ViewController.swift
+//  tvOSMultiplePlayersTemplate
+//
+//  Created by Aleksandr Pavliuk on 6/13/17.
+//
+//
+
+import UIKit
+
+class ViewController: UIViewController {
+
+    @IBOutlet weak var onePlayerButton: UIButton!
+    @IBOutlet weak var fivePlayersButton: UIButton!
+    @IBOutlet weak var tenPlayersButton: UIButton!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+    }
+    
+    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
+        guard let playersCollectionViewController = segue.destination as? PlayersCollectionViewController, let button = sender as? UIButton else { return }
+        
+        
+        playersCollectionViewController.playersQuantity = button.tag
```

Comment:
Using `button.tag` as a data transfer object is a bad idea. Instead it should be explicitly and clearly specified.